app/src/main/java/com/plan2meet/dto/EventDTO.kt

```diff
@@ -14,4 +17,12 @@ class EventDTO (
     override fun toString(): String {
         return startTime.toString()
     }
+
+    @JvmField
+    @PrimaryKey(autoGenerate = true)
+    var id: Long = 0
+
+    fun UpdateDb(){
```

Comment:
An Update Db function could be created here which can use the database manager class

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.